### PR TITLE
Speech to text

### PR DESCRIPTION
**Pull request by Amit366**
## Related Issue
  Added Speech to text converter

Closes: #872 

### Describe the changes you've made
Added a code that converts speech to text.

### Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] New and existing unit tests pass locally with my changes.

### Screenshots
Screenshot (180)

### CODE REVIEW

**Comment by Amit366** [comment]
@ridsuteri please review it.

File: script.js [hunk 1]
```diff
@@ -555,12 +555,21 @@ let projectData = [
     projectImage: "assets/GIFs/FallingBulletsAnimation.gif",
     projectUrl: "public/FallingBulletsAnimation.html",
   },
+  {
+    projectName: "Speech To Text Converter",
+    projectImage: "assets/GIFs/SpeechtoText.gif",
+    projectUrl: "public/SpeechtoText.html",
+  }
```
Comment (ridsuteri) [inline script.js:562]: , is missing after line 562
please add it

**Review by ridsuteri — CHANGES_REQUESTED** [review]
there is a little fix needed at line 562 in script.js ,please resolve it

**Comment by Amit366** [comment]
@ridsuteri now is it fine??

**Comment by Amit366** [comment]
@ridsuteri please review it. It is getting merge conflicts.

**Review by ridsuteri — APPROVED** [review]
good work @Amit366 thanks for contributing :)

**Pull request merged**